docs/level1/02-draw-all-elements-from-response.md

```diff
@@ -1,6 +1,7 @@
 ## Step 2
 
-Replace the last `then` callback content to print out all the items in the database using `forEach`
+Replace the content of the last `then` callback to print out all the items we have.
+We do this with the the help of the `forEach` funciton
```

Comment:
thanks for catching the typo.
Not sure about "method" vs "function", there is a subtle (imo almost irrelevant) difference.
I'd try to be consistent with the rest of the write-up instead of introducing new words.